crt_portal/crt_portal/settings.py

```diff
@@ -335,8 +335,7 @@
     ('css', 'compressor_toolkit.precompilers.SCSSCompiler'),
 )
 
-# would like to add this before public release
-COMPRESS_ENABLED = False
+COMPRESS_ENABLED = True
```

Comment:
I considered disabling this for local dev usage, but I like the increased rendering speed. Might cause some issues when debugging, but open to suggestions here.